includes/assets/js/activitypub.js

```diff
@@ -0,0 +1,52 @@
+(function($) {    
```

Comment:
thanks for moving, but I would place the `assets` folder in the root folder, not inside the `includes` folder.